Add "What is "publish size" vs "install size"?" section (#2)
antonk52/why-npm-i-so-long: README.md

Little utility to ease troubleshooting why installing npm dependencies takes too long.

## What is "publish size" vs "install size"?

> The "publish size" is the size of the source code published to npm. This number is easy to detect and is typically very small.

Reviewer: ```suggestion
The "publish size" is the size of the source code published to npm. This number is easy to detect and should be pretty small.
```
